.markdownlint-cli2.yaml

```diff
@@ -0,0 +1,4 @@
+config:
```

Comment:
https://github.com/DavidAnson/vscode-markdownlint/blob/55658e4b011caa72440ce143d88936ee566386dc/markdownlint-cli2-config-schema.json is the schema, but I don't want to introduce redhat yaml language server for now(I dropped it past)